projects/hello-world/_posts/2020-09-30-tex.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Hello World in Tex
+layout: default
+last-modified: 2020-09-30
+tags: [tex, hello-world]
+authors:
+  - bracciata
```

Comment:
Make sure your name is in the authors file: https://github.com/TheRenegadeCoder/sample-programs-website/blob/master/_data/authors.yml

Reply:
Should I do my full name or just my handle?

Reply:
On it!

Reply:
Made a new pull request doing so #321